Makefile

```
@@ -3,6 +3,7 @@ TOOLS_DIR := tools
 
 BABYLON_PKG := github.com/babylonchain/babylon/cmd/babylond
 WASMD_PKG   := github.com/CosmWasm/wasmd/cmd/wasmd
+BCD_PKG     := github.com/babylonchain/babylon-sdk/cmd/bcd
```

```suggestion
BCD_PKG     := github.com/babylonchain/babylon-sdk/demo/cmd/bcd
```

CI failed due to this line. Also we need to add 

```go
import (
	_ "github.com/babylonchain/babylon/cmd/babylond"
	_ "github.com/CosmWasm/wasmd/cmd/wasmd"
	_ "github.com/babylonchain/babylon-sdk/demo/cmd/bcd"
)
```

in `tools/tools.go` so that `tools/` package can recognise the `wasmd`/`bcd` packages